frontend/src/components/V4Button.vue

```diff
@@ -114,9 +135,9 @@ button.v4-button.icon-button {
    border: 1px solid var(--uvalib-grey);
    color: black;
 }
-.pure-button.pure-button-tertiary:focus {
-   box-shadow: 0 0 0 4px rgba(150,150,150, 0.3); 
-}
+// .pure-button.pure-button-tertiary:focus {
```

Comment:
the tertiary focus was removed and not replaced. When opening the popovers like What am I searching, you can't tell that the close button is focused

Reply:
thanks - fixing now